tests/passes/synthesis/test_qsearch.py

```diff
@@ -30,3 +29,25 @@ def test_small_qutrit(self) -> None:
         circuit.perform(leap)
         dist = circuit.get_unitary().get_distance_from(utry, 1)
         assert dist <= leap.success_threshold
+
+    def test_seed(self) -> None:
+        seed = Circuit(2)
+        seed.append_gate(CNOTGate(), (0, 1))
+        layer_gen = SeedLayerGenerator(seed)
+        qsearch = QSearchSynthesisPass(layer_generator=layer_gen)
+        utry = UnitaryMatrix.random(2)
+        circuit = Circuit.from_unitary(utry)
+        circuit.perform(qsearch)
```

Comment:
We shouldn't use `circuit.perform` in tests for startup times. You can use the pytest fixture `compiler: Compiler`.

Reply:
For `test_seed_in_data`, can I explicitly provide a `PassData` object using `compiler.compile`?

Reply:
Theres a pass that explicited updates a key value pair in data. I forget the name

Reply:
Found it! It's the aptly named `UpdateDataPass`